package.json

```diff
@@ -2239,7 +2292,7 @@
         "tslint-microsoft-contrib": "^5.0.3",
         "tslint-plugin-prettier": "^2.1.0",
         "typemoq": "^2.1.0",
-        "typescript": "^4.4.3",
+        "typescript": "^4.5.4",
```

Comment:
THis seems like an odd addition? Did you need a new typescript compiler?

Reply:
No need for a new TS compiler. I think it just updated it with the one I used.

Reply:
You should back out this change then. This should also remove the change to package-lock.json